### PR TITLE
Disables link button when image is selected.

### PR DESCRIPTION
**Pull request by eireland**
This adds css to show that linking is disabled when selected item is an image.

### CODE REVIEW

File: src/editor-toolbar/toolbar-button.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const ToolbarButton: React.FC<IProps> = (props: IProps) => {
   const onChangeProps = onChange ? { onChange } : {};
   const iconProps = { width: iconSize, height: iconSize, fill, ...onChangeProps };
   return (
-    <div className="toolbar-button" style={buttonStyle} title={tooltip}
+    <div className={`toolbar-button ${!isEnabled && format === "link" ? "disabled" : ""}`} style={buttonStyle} title={tooltip}
```
[the remark this answers is not in the document]
Comment (eireland) [reply src/editor-toolbar/toolbar-button.tsx:81]: Without it, all the toolbar buttons become disabled.

[the remark this answers is not in the document]
Comment (kswenson) [reply src/editor-toolbar/toolbar-button.tsx:81]: Looking at lines 76-77, the issue is that `isEnabled` must be explicitly `false`, so the correct formulation would then be:
```suggestion
    <div className={`toolbar-button ${isEnabled === false ? "disabled" : ""}`} style={buttonStyle} title={tooltip}
```

**Review by kswenson — CHANGES_REQUESTED** [review]
Closer...

**Review by kswenson — APPROVED** [review]
👍 Looks good

**Pull request merged**